components/03-modules/toolbar/_toolbar.scss

```diff
@@ -221,4 +243,23 @@ $toolbar__select-dropdown-border-radius: 0 0 $spacer--medium $spacer--medium !de
             background-position: center center;
         }
     }
+
+    .pager {
+        display: none;
+    }
+
+    ~ .toolbar {
```

Comment:
`& ~ .toolbar`

Reply:
Isn't better to keep it simple? I think ampersand is unnecessary here, like [there](https://css-tricks.com/the-sass-ampersand/#article-header-id-4)